### ISSUE TITLE
OM::XML::TemplateMissingException for JPEG 2000 characterization

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/curation_concerns#237]
Caused by setting filename fails with OM::XML::TemplateMissingException if characterization doesn't work or fits is not present.
https://github.com/projecthydra-labs/curation_concerns/blob/master/curation_concerns-models/app/services/curation_concerns/characterization_service.rb#L19

**Comment by jcoyne** [comment]
```
CurationConcerns::FullTextExtractionService.run(gf)
=> ""
```

**Comment by jcoyne** [comment]
I don't think that `ng_xml=` accepts a string such as:

https://github.com/projecthydra-labs/curation_concerns/blob/master/curation_concerns-models/app/services/curation_concerns/characterization_service.rb#L35

**Comment by jcoyne** [comment]
We were actually running into this: https://github.com/harvard-lts/fits/issues/20
The xml returned from FITS was invalid.

**Comment by jcoyne** [comment]
Seems like upgrading FITS from 0.6.0 to 0.6.3 fixes this.

**Comment by jcoyne** [comment]
```ruby
gf = GenericFile.find('zk51vg76b')

 char = Hydra::FileCharacterization.characterize(gf.original_file.content, gf.original_file.original_name, :fits) { |config| config[:fits] = Hydra::Derivatives.fits_path }
"READBOX seen=true\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<fits xmlns=\"http://hul.harvard.edu/ois/xml/ns/fits/fits_output\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd\" version=\"0.6.2\" timestamp=\"8/31/15 4:01 PM\">\r\n  <identification status=\"CONFLICT\">\r\n..."
```

**Comment by jcoyne** [comment]
Closed by hydra-file_characterization 0.3.2
